packages/executable/package.json

```diff
@@ -13,15 +13,19 @@
   "pkg": {
     "assets": [
       "../client/build/**/*",
-      "../server/default_img/**/*"
+      "../server/**/*"
     ],
     "scripts": [
       "../server/addons/**/*"
-    ]
+    ],
+    "targets": [
+      "node14-macos-x64"
+    ],
+    "outputPath": "build"
   },
   "scripts": {
-    "start": "node helpers/pkgResolveErrors.js && pkg . --out-path ./builds && node helpers/pkgCopyNodes.js",
-    "pkg-build": "pkg . --out-path ./builds"
+    "start": "node helpers/pkgResolveErrors.js && rm -rf build && pkg . && node helpers/pkgCopyNodes.js",
+    "pkg-build": "rm -rf build && pkg ."
```

Comment:
@lebaphi - we don't need the script at line 28, do we?

Reply:
yep, I think we don't need it. I see it existed before so I update it again :D